### PR TITLE
New UI changes

### PR DESCRIPTION
**Pull request by duncanleo**
(according to Jurvis' design)

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
         <activity
             android:name=".activities.MainActivity"
             android:label="@string/app_name"
-            android:theme="@style/AppTheme.NoActionBar"/>
+            android:theme="@style/AppTheme"/>
```
Comment (mananwason) [inline app/src/main/AndroidManifest.xml]: Why this??

File: app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 2]
```diff
@@ -59,15 +70,19 @@
...
+//    private DrawerLayout mDrawerLayout;
 
     private Toolbar mToolbar;
 
-    private NavigationView navigationView;
+//    private NavigationView navigationView;
+//
+//    private ProgressBar downloadProgress;
+//
+//    private CoordinatorLayout mainFrame;
```
Comment (mananwason) [inline app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: Commented code? Please remove it.

File: app/src/main/java/org/fossasia/openevent/activities/MainActivity.java [hunk 3]
```diff
@@ -375,7 +394,7 @@ public void onEventsDownloadDone(EventDownloadEvent event) {
             }
         } else {
 
-            downloadFailed();
+//            downloadFailed();
```
Comment (mananwason) [inline app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: How do you plan to see if the downloadfailed?

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (duncanleo) [inline app/src/main/AndroidManifest.xml]: The new design has an action bar.

File: app/src/main/java/org/fossasia/openevent/activities/MainActivity.java (hunk 2 above)
Comment (duncanleo) [inline app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: I'm still in the process of migrating the code for the old design to the new one

File: app/src/main/java/org/fossasia/openevent/activities/MainActivity.java (hunk 3 above)
Comment (duncanleo) [inline app/src/main/java/org/fossasia/openevent/activities/MainActivity.java]: I'm still in the process of migrating the code to fit the new views etc used by the new design

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (mananwason) [inline app/src/main/AndroidManifest.xml]: Why. We are already using toolbars!!

Comment (duncanleo) [inline app/src/main/AndroidManifest.xml]: @mananwason I think I'll send another pull request when all these has been cleared up

**Comment by mananwason** [comment]
@duncanleo This PR is incomplete and seriously I don't see any design changes. Instead some of the things have been broken for instance there is a gap between action bar and progress bar.

**Pull request closed without merging**